cmd/web/main.go

```diff
@@ -6,35 +6,48 @@ import (
   "crypto/tls"
   "os"
   "net/http"
+  "html/template"
   )
 
+//created type application in order to facilitate passing data for sessions
+type application struct {
+  infoLog          *log.Logger
```

Comment:
The STDLib logger is fine, but take a look at logrus or zap. I enjoy zap myself. It provides helper functions to include data structures etc without formatting the output line manually.

Reply:
Thanks, anything to make the application lighter weight!